packages/upswyng-web/src/components/ReportIssue.tsx

```diff
@@ -0,0 +1,194 @@
+      checked: false,
+    },
+    website: {
+      text: "The website link is wrong.",
+      checked: false,
+    },
+    schedule: {
+      text: "The schedule is wrong.",
+      checked: false,
+    },
+    directions: {
+      text: "The directions are wrong.",
+      checked: false,
+    },
+    other: {
+      text: "Other",
+      checked: false,
+    },
+  } as State);
+
+  const selectedIssues = (): string => {
+    const selected: string[] = [];
+    Object.keys(issues).forEach(issue => {
+      if (issues[issue].checked) {
+        selected.push(issues[issue].text);
+      }
+      if (selected.includes("Other")) {
+        selected.pop();
+        if (other) {
+          selected.push(other);
+        }
+      }
+    });
+    return JSON.stringify(selected);
+  };
+
+  const handleSubmit = () => {
+    axios
+      .post(`${serverUri}/api/resources/issues/user-report`, {
+        resourceId: resourceId,
+        detailExplanation: comments,
+        reportedIssues: selectedIssues(),
+      })
+      .then(() => setFormSubmitted(true))
+      .catch(() => setError(true));
+  };
+
+  const handleCheck = (key: string) => (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    updateIssues({
+      ...issues,
+      [key]: { ...issues[key], checked: event.target.checked },
+    });
+  };
+
+  if (resource === undefined) {
+    return <LoadingSpinner />;
+  }
+
+  return (
+    <Container>
+      <PageBanner
+        text="Report an Issue"
+        backButtonAction={lastLocation ? history.goBack : null}
+      />
+      <Typography component="div">
+        Report an issue for:{" "}
+        <Typography color="primary">{resource?.name}</Typography>
+      </Typography>
+      {formSubmitted ? (
+        <>
+          <Typography color="secondary" className={classes.marginTop}>
+            The problem has been submitted and will be reviewed. Thank you for
+            your feedback!
+          </Typography>
+          <Button
+            fullWidth
+            variant="contained"
+            color="primary"
+            onClick={history.goBack}
+            className={classes.marginTop}
+            size="medium"
+          >
+            Back
+          </Button>
+        </>
+      ) : (
+        <>
+          <FormGroup className={classes.marginTop}>
+            {Object.keys(issues).map((issue: string) => {
+              const { text, checked } = issues[issue];
+              return (
+                <FormControlLabel
+                  control={
+                    <Checkbox checked={checked} onChange={handleCheck(issue)} />
+                  }
+                  label={text}
+                  key={text}
+                />
+              );
+            })}
+            <Input
+              disabled={!issues.other.checked}
+              value={issues.other.checked ? other : ""}
+              placeholder="Explain..."
+              onChange={e => setOther(e.target.value)}
+            />
```

Comment:
We should go over some of the styles tonight. Our MUI theme probably needs updated to discern between a disabled and enabled text input.